test/e2e_node/jenkins/e2e-node-jenkins.sh

```diff
@@ -28,7 +28,16 @@ set -x
 
 . $1
 
+if [ "$INSTALL_GODEP" = true ] ; then
```

Comment:
i just went ahead and removed this whole step.  the new e2e-node-jenkins doesnt need this, it used to require it for some reason